.github/workflows/sival_nightly.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Nigthly
+on:
+  schedule:
+    - cron: "00 04 * * *"
+
+jobs:
+  FPGA-CW310-SiVal-Nightly:
+    name: FPGA CW310 SiVal nightly tests
+    runs-on: [ubuntu-20.04-fpga, cw310]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Required for the bitstream cache work.
+          ref: earlgrey_1.0.0 # schedule only work on the default branch, but we want to run another branch.
+
+      - uses: ./.github/actions/install-deps
+      - name: ROM boot stage
+        run: |
+          module load xilinx/vivado
+          ./bazelisk.sh sync --configure
```

Comment:
I don't think you need to do this if it's the first time the command runs, only if you change branches afterwards